esteid/idcard/README.md

```diff
@@ -0,0 +1,50 @@
+# Signing with ID Card
+
+## Process outline
+
+1. Fetch the user certificate from the ID Card
+
+   On the frontend, query the the `hwcrypto.js` API that interacts with the ID card,
+   to obtain the certificate that would be required for the initial XAdEs structure.  
+
+1. Initialization request to the backend 
+
+   The backend creates a BDoc (Asic-E) container with files to be signed (or takes an existing container). 
+   Employing the user certificate received from the ID card on step 1, it creates a XAdEs structure 
```

Comment:
```suggestion
   Using the user certificate received from the ID card on step 1, it creates a XAdEs structure 
```

Reply:
I rephrased it a little to be even more clear :)